### ISSUE TITLE
store.describe_data(data_id) does not always return a time_range with values 

### ISSUE THREAD

**Issue by AliceBalfanz** [dcs4cop/xcube#428]
**Describe the bug**

When using the xcube python api and using describe_data for a certain store, the time_range is not always filled with values

**To Reproduce**
```
from xcube.core.store import new_data_store
store = new_data_store('s3', bucket_name='cop-services')
from IPython.display import JSON

store = new_data_store('s3', bucket_name='cop-services')
JSON(list(store.get_data_ids()))
data_id = 'OCEANCOLOUR_BAL_CHL_L3_NRT_OBSERVATIONS_009_049.zarr'
store.describe_data(data_id)
```
![image](https://user-images.githubusercontent.com/23457217/113736155-282f1a80-96fd-11eb-8d8c-1bd8baef48a9.png)

**Expected behavior**
I would expect at least a start time, in case the dataset is continuously growing.

**Comment by TonioF** [comment]
Should be solved with https://github.com/dcs4cop/xcube/pull/418

**Comment by forman** [comment]
> Should be solved with #418

Confirmed:

![image](https://user-images.githubusercontent.com/206773/116980305-5e44c780-acc6-11eb-9125-647bb4279634.png)
